commands/show/nodepool/command.go

```diff
@@ -134,13 +134,13 @@ func fetchNodePool(args arguments) (*result, error) {
 		return nil, microerror.Mask(err)
 	}
 
-	res.instanceTypeDetails, err = awsInfo.GetInstanceTypeDetails(res.nodePool.NodeSpec.Aws.InstanceType)
-	if err != nil {
-		return nil, microerror.Mask(err)
-	}
+	var nodespecErr error
+	res.instanceTypeDetails, nodespecErr = awsInfo.GetInstanceTypeDetails(res.nodePool.NodeSpec.Aws.InstanceType)
 
-	res.sumCPUs = res.nodePool.Status.NodesReady * int64(res.instanceTypeDetails.CPUCores)
-	res.sumMemory = float64(res.nodePool.Status.NodesReady) * float64(res.instanceTypeDetails.MemorySizeGB)
+	if nodespecErr == nil {
```

Comment:
We should differentiate between expected and unexpected errors to make the behaviour and experience for the user more predictable and reliable at the end of the day. With this there might be something going on that we miss in the first place only because the error handling is insufficient.

Reply:
@xh3b4sd Please check whether https://github.com/giantswarm/gsctl/pull/381/commits/b40efd4409db86a76f5863c848f9e8a352c02f37 fixes this for you. Thanks!